src/GovUk.Education.ExploreEducationStatistics.Common/Cache/CacheAttribute.cs

```diff
@@ -11,22 +16,20 @@ namespace GovUk.Education.ExploreEducationStatistics.Common.Cache
     /// Base caching attribute that should be extended with specific
     /// caching implementations e.g. blob storage, memory, Redis, etc.
     /// </summary>
-    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    [AttributeUsage(AttributeTargets.Method)]
     [Injection(typeof(CacheAspect), Inherited = true)]
-    public abstract class CacheAttribute : Attribute
+    public abstract class CacheAttribute : BaseUniversalWrapperAttribute
     {
+        private const BindingFlags ConstructorBindingFlags =
+            BindingFlags.Instance | BindingFlags.Public | BindingFlags.CreateInstance |
+            BindingFlags.OptionalParamBinding;
+
         /// <summary>
         /// The base type of the cache key to use.
         /// The <see cref="Key"/> must be assignable to this.
         /// </summary>
         protected virtual Type BaseKey => typeof(ICacheKey);
 
-        /// <summary>
-        /// Cache with highest priority is checked first.
-        /// 0 - Lowest, 255 - Highest.
-        /// </summary>
-        public byte Priority { get; init; } = 0;
-
```

Comment:
I discussed this with Dunc. I won't speak for Dunc, but after that conversation my thought was as it wasn't being used, we can remove it and create a ticket to add it again if we feel strongly about it.

I have also wondered if we can basically replicate the same effect - not sure if this would actually do the same job though:
```
[MemoryCache]
public T method1()
{
  method2();
}

[BlobCache]
public T method2()
{
  DoStuff();
}
```

Reply:
I started implementing this, and I think I've spotted a bug with the existing code.

I believe the bug is around how `ForceUpdate` interacts with multiple cacheTriggers. If you have multiple triggers, the first priority being `MemoryCache` with `ForceUpdate` set, and the second being `BlobCache` with a cached value, currently, the `MemoryCache` wouldn't get updated.

I think there is some ambiguity around what we would expect `ForceUpdate` to do in different situations, so if `MemoryCache` is prioritised and has a cached value, but the lower priority `BlobCache` does have `ForceUpdate` set, we will probably want to update the `BlobCache` cached value, but should we just return the `MemoryCache` cached value, or should that be updated as well?

Rather than try to address this in this PR, I'm going to create another ticket for it.

Reply:
Yeh I was alluding that there are potential issues with the existing code, so it's fine to leave this to another ticket!